nsv13/code/game/objects/items/religion.dm

```diff
@@ -0,0 +1,24 @@
+/obj/item/banner/munitions
+	name = "people's republic of munitions banner"
+	desc = "the banner of the people's republic of munitions, masters of arms and ammunition"
+	icon = 'nsv13/icons/obj/banners.dmi'
+	icon_state = "banner_munitions"
+	item_state = "banner_munitions"
+	lefthand_file = 'nsv13/icons/mob/inhands/equipment/banners_lefthand.dmi'
+	righthand_file = 'nsv13/icons/mob/inhands/equipment/banners_righthand.dmi'
+	job_loyalties = list("Master At Arms", "Munitions Technician", "Flight Leader", "Fighter Pilot", "Air Traffic Controller")
```

Comment:
"Flight Leader" and "Fighter Pilot" are not valid jobs.

Reply:
Updated the jobs.